pom.xml

```diff
@@ -174,6 +174,11 @@
         <artifactId>junit</artifactId>
         <version>4.11</version>
         <scope>test</scope>
+      </dependency>
+	  <dependency>
+        <groupId>org.assertj</groupId>
+		<artifactId>assertj-core</artifactId>
+		<version>2.6.0</version>
```

Comment:
You use it only in the test case, right? If so, should you specify the scope?

```xml
<scope>test</scope>
```
Or is it not possible to deal with existing assert libraries? I do not want to increase dependency unnecessarily.

Reply:
@daisuke-yoshimoto thanks a lot for your feedback in this PR, can you please make sure that you notify @balsarori when this is ready to merge?